scripts/neighbor_advertiser

```diff
@@ -383,30 +382,48 @@ def reset_vxlan_tunnel():
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('-s', dest='vip', metavar='vip', type = str, required = True, help = 'ferret service vip')
+    parser.add_argument('-s', dest='vips', metavar='vips', type = str, required = False, help = 'ferret service vip list, required in set mode')
     parser.add_argument('-m', dest='mode', metavar='mode (set, reset)', type = str, required = True, choices=['set', 'reset'], help = 'operation mode')
     args = parser.parse_args()
 
-    ferret_service_vip = args.vip
+    ferret_service_vips = args.vips
     operation_mode = args.mode
 
+    if operation_mode == 'set' and ferret_service_vips == None:
+        log_warning('ferret service vip is required in set mode')
+        return -1
+
     connect_config_db()
 
     if operation_mode == 'set':
-        ferret_server_ip = post_neighbor_advertiser_slice(ferret_service_vip)
-        set_vxlan_tunnel(ferret_server_ip)
-        set_mirror_tunnel(ferret_server_ip)
+        set_success = False
+
+        for ferret_service_vip in ferret_service_vips.split(','):
+            ferret_server_ip = post_neighbor_advertiser_slice(ferret_service_vip)
+
+            if ferret_server_ip == None:
+                continue
+            else:
```

Comment:
The 3 lines above can be condensed to "if ferret_server_ip is not None:

Reply:
updated, thx